feddlib/core/LinearAlgebra/Matrix_decl.hpp

```diff
@@ -61,6 +61,13 @@ class Matrix {
     typedef Teuchos::Comm<int> Comm_Type;
     typedef Teuchos::RCP<const Comm_Type> CommConstPtr_Type;
 
+    typedef Xpetra::Import<LO,GO,NO> XpetraImport_Type;
+    typedef Teuchos::RCP<XpetraImport_Type> XpetraImportPtr_Type;
+
+    typedef Xpetra::Export<LO,GO,NO> XpetraExport_Type;
+    typedef Teuchos::RCP<XpetraExport_Type> XpetraExportPtr_Type;
+
+
```

Comment:
Could you add some comments, at least, for the new functions?

Reply:
Will do.